font-codegen/src/parsing.rs

```diff
@@ -270,7 +270,7 @@ impl InlineExpr {
 pub(crate) enum FieldType {
     Offset {
         typ: syn::Ident,
-        target: Option<syn::Ident>,
+        target: Option<OffsetTarget>,
```

Comment:
I'm struggling to grasp what an Offset without a target means, perhaps a comment explaining why it's not `target: OffsetTarget`?

Reply:
This is a fairly little-used codepath, but sometimes the target of an offset is not a type we can easily represent in codegen (for instance in the top-level table directory, where the offset can be any different table) or is a type that we haven't implemented yet.

Reply:
(I did a little audit: this is used in a few places where there are variation tables we don't support yet, intentionally in the root `TableDirectory`, and by accident in cmap14)

Reply:
The "the target of an offset..." explanation would make a magnificent comment. Definitely would have helped me as reader.